Tests/RevenueCatUITests/PaywallsV2/VariableHandlerV2Tests.swift

```diff
@@ -64,7 +64,58 @@ class VariableHandlerV2Test: TestCase {
             "num_year_two": "%d years",
             "num_year_few": "%d years",
             "num_year_many": "%d years",
-            "num_year_other": "%d years"
+            "num_year_other": "%d years",
+            "num_days_short": "%dd",
+            "num_weeks_short": "%dwk",
+            "num_months_short": "%dmo",
+            "num_years_short": "%dyr"
+        ],
+        "ar": [
```

Comment:
Hmm FFTI but should the values in this other locale be different than the `en_US` values? I guess this is mostly testing variable handling and not anything else, but might help making sure there isn't an issue during tests... NABD though, happy to leave it as is.